docs/examples/code-cells.ipynb

```diff
@@ -0,0 +1,960 @@
+    "By default, plots are generated in the PNG format.\n",
+    "In most cases,\n",
+    "it looks better\n",
+    "if SVG plots are used for HTML output\n",
+    "and PDF plots are used for LaTeX/PDF.\n",
+    "This can be achieved by setting\n",
+    "[nbsphinx_execute_arguments](https://nbsphinx.readthedocs.io/en/0.9.7/configuration.html#nbsphinx_execute_arguments)\n",
+    "in your `conf.py` file like this:\n",
+    "\n",
+    "```python\n",
+    "nbsphinx_execute_arguments = [\n",
+    "    \"--InlineBackend.figure_formats={'svg', 'pdf'}\",\n",
+    "]\n",
+    "```\n",
+    "    \n",
+    "In the following example, `nbsphinx` should use an SVG image in the HTML output\n",
+    "and a PDF image for LaTeX/PDF output\n",
+    "(other Jupyter clients like JupyterLab will still show the default PNG format)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots(figsize=[6, 3])\n",
+    "ax.plot([4, 9, 7, 20, 6, 33, 13, 23, 16, 62, 8]);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For comparison,\n",
+    "this is how it would look in PNG format ..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%config InlineBackend.figure_formats = ['png']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "... and in `'png2x'` (a.k.a. `'retina'`) format:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%config InlineBackend.figure_formats = ['png2x']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Pandas Dataframes\n",
+    "\n",
+    "[Pandas dataframes](https://pandas.pydata.org/pandas-docs/stable/user_guide/dsintro.html#dataframe)\n",
+    "should be displayed as nicely formatted HTML tables (if you are using HTML output)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.DataFrame(\n",
+    "    np.random.randint(0, 100, size=[10, 4]),\n",
+    "    columns=[r\"$\\alpha$\", r\"$\\beta$\", r\"$\\gamma$\", r\"$\\delta$\"],\n",
+    ")\n",
+    "df"
+   ]
```

Comment:
output is illegible in dark mode:

![Screenshot_2025-05-08_11-24-49](https://github.com/user-attachments/assets/d5bb841a-434c-4c79-b66c-9e39fee5c0b0)